src/interface_adapter/rooms/RoomsPresenter.java

```diff
@@ -0,0 +1,25 @@
+package interface_adapter.rooms;
+
+import use_case.rooms.RoomsOutputBoundary;
+import use_case.rooms.RoomsOutputData;
+
+public class RoomsPresenter implements RoomsOutputBoundary {
+  private final RoomsViewModel roomsViewModel;
+
+  public RoomsPresenter(RoomsViewModel roomsViewModel) {
+    this.roomsViewModel = roomsViewModel;
+  }
+
+  @Override
+  public void prepareSuccessView(RoomsOutputData response) {
```

Comment:
Thinking aloud here, I think we might want a "LoadingPresenter" that is embedded on every page with data from firebase. (This would be done in a future PR)